Formula/corectl.rb

```diff
@@ -42,6 +42,7 @@ def install
 
       bin.install Dir["bin/*"]
 
+      doc.install "README.md", "LICENSE"
```

Comment:
@AntonioMeireles This should not be here. Those go in the prefix and are handled automatically: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/metafiles.rb